minetest.conf.example

```diff
@@ -151,6 +151,10 @@
 #enable_3d_clouds = true
 #    Use a cloud animation for the main menu background
 #menu_clouds = true
+#    Enable random menu image, the subgame must support this by having multiple
+#    images in the menu folder, named "background (or overlay).png.x" where x
+#    is a number, increasing from 1, with a maximum of 9.
+#random_menu_images = false
```

Comment:
Other features that randomize the media used do not need a setting either, it would be better if this didn't need a setting either; how this could be implemented in Lua: (untested)
```lua
local count = 1
for i = 9, 1, -1 do
    local f = io.open(path .. "." .. i,"rb")
    if f ~= nil then
        count = i
        io.close(f)
        break
    end
end
local n = math.random(1, count)
local new_path = path .. "." .. n
if n == 1 then -- Use background.png instead of background.png.1
    new_path = path
end
path = new_path
```